diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter5/LambdaLengthWarnTest.kt

```diff
@@ -0,0 +1,158 @@
+package org.cqfn.diktat.ruleset.chapter5
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.LambdaLengthRule
+import org.cqfn.diktat.util.LintTestBase
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class LambdaLengthWarnTest : LintTestBase(::LambdaLengthRule) {
```

Comment:
Please add tests with case that we discovered: when there is lambda which doesn't expect parameters and when there is a nested lambda with implicit parameter.